sdks/python/apache_beam/ml/transforms/embeddings/vertex_ai_test.py

```diff
@@ -245,5 +248,45 @@ def test_mltransform_to_ptransform_with_vertex(self):
           ptransform_list[i]._model_handler._underlying.model_name, model_name)
 
 
+@unittest.skipIf(
+    VertexAIImageEmbeddings is None, 'Vertex AI Python SDK is not installed.')
+class VertexAIImageEmbeddingsTest(unittest.TestCase):
```

Comment:
I am open to changing this to an integration test since we're accessing a GCS bucket; however, we aren't actually loading the image on-machine and that load is happening on the vertex side

Reply:
Maybe create a directory `test_resources`  or something similar and put the image there instead of GCS?